### PR TITLE
python310Packages.py-sr25519-bindings: 0.2.0 -> unstable-2023-03-15

### PR DESCRIPTION
**Pull request by jvanbruegge**
###### Description of changes

ZHF: https://github.com/NixOS/nixpkgs/issues/230712

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/py-sr25519-bindings/default.nix [hunk 1]
```diff
@@ -11,26 +11,24 @@
...
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "polkascan";
     repo = "py-sr25519-bindings";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-Lu3J0+YeQHHKItOZTT24DlQAUJuE9fd+py6Eb46/MSE=";
+    rev = "9127501235bf291d7f14f00ec373d0a5000a32cb";
```
Comment (RaitoBezarius) [inline pkgs/development/python-modules/py-sr25519-bindings/default.nix:22]: This is not a build fix, this is a bump to a unstable revision, for which reason?

Comment (jvanbruegge) [reply pkgs/development/python-modules/py-sr25519-bindings/default.nix:22]: Sorry, the commit message was wrong. This commits fixes the build in upstream

File: pkgs/development/python-modules/py-sr25519-bindings/default.nix [hunk 2]
```diff
@@ -11,26 +11,24 @@
 
 buildPythonPackage rec {
   pname = "py-sr25519-bindings";
-  version = "0.2.0";
+  version = "2023-03-15-unstable";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/py-sr25519-bindings/default.nix:14]:
```suggestion
  version = "unstable-2023-03-15";
```

Comment (SuperSandro2000) [inline pkgs/development/python-modules/py-sr25519-bindings/default.nix:14]:
```suggestion
  version = "unstable-2023-03-15";
```

Comment (jvanbruegge) [reply pkgs/development/python-modules/py-sr25519-bindings/default.nix]: done

**Comment by mweinelt** [comment]
The attribute in the commit message must be `python310Packages.py-sr25519-bindings`.

@ofborg build python310Packages.py-sr25519-bindings
@ofborg build python311Packages.py-sr25519-bindings

File: pkgs/development/python-modules/py-sr25519-bindings/default.nix [hunk 3]
```diff
@@ -11,26 +11,24 @@
...
+    rev = "9127501235bf291d7f14f00ec373d0a5000a32cb";
+    hash = "sha256-mxNmiFvMbV9WQhGNIQXxTkOcJHYs0vyOPM6Nd5367RE=";
   };
 
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-MwJr6D1akKc+ImCTn4fj8fRuL+PdCmv39usMmcEvPa0=";
```
Comment (mweinelt) [inline pkgs/development/python-modules/py-sr25519-bindings/default.nix:29]: Getting a different hash here.

         specified: sha256-MwJr6D1akKc+ImCTn4fj8fRuL+PdCmv39usMmcEvPa0=
            got:    sha256-7fDlEYWOiRVpG3q0n3ZSS1dfNCOh0/4pX/PbcDBvoMI=

Comment (SuperSandro2000) [reply pkgs/development/python-modules/py-sr25519-bindings/default.nix:29]:
```suggestion
    hash = "sha256-7fDlEYWOiRVpG3q0n3ZSS1dfNCOh0/4pX/PbcDBvoMI=";
```

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.py-sr25519-bindings

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**